packages/nuxt/src/core/nitro.ts

```
@@ -249,7 +249,8 @@ export async function initNitro (nuxt: Nuxt & { _nitro?: Nitro }) {
     const buildId = nuxt.options.runtimeConfig.app.buildId ||= nuxt.options.buildId
     const buildTimestamp = Date.now()
 
-    const manifestPrefix = joinURL(nuxt.options.app.buildAssetsDir, 'builds')
+    const baseURL = nuxt.options.runtimeConfig.app.baseURL ||= nuxt.options.app.baseURL
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)